package.json

```diff
@@ -16,7 +16,7 @@
   },
   "bin": "dist/create-originate-app.js",
   "scripts": {
-    "dev": "ts-node src/create-originate-app.ts",
+    "dev": "NODE_ENV=test ts-node src/create-originate-app.ts",
```

Comment:
@hallettj on a scale of 1-10 how sad does this make you. :)

Reply:
Eh, it's fine